src/api/getWilayaByPhoneCode.js

```diff
@@ -9,15 +9,17 @@ const getWilayaByPhoneCode = data =>
 * //returns {mattricule: '6', name: 'Béjaïa', ...} because the phone code 34 belongs to Béjaïa
 * getWilayaByPhoneCode(34)
 *
-* @param { Number | String } phoneCode is a number if the latter corresponds to the wilaya's phone code and a string if's the full phone number
+* @param { Number } phoneCode the phone code
 * @param {String[]} projection a list of wilaya object attributes to keep
 * @returns { Object | undefined } Returns the target object, or undefined
 */
   (phoneCode, projection) => {
-    const phoneCodeVar =  typeof phoneCode === "number"
-      ? phoneCode
-      : parseInt(phoneCode.substring(1, 3), 10);
-    const wilaya = data.find(w => w.phoneCodes.includes(phoneCodeVar));
+    if(!Number.isInteger(phoneCode)){
+      return;
+    }
+    const wilaya = data.find(w => w.phoneCodes.includes(phoneCode));
```

Comment:
What do think of this solution?

```suggestion
    if(Number.isNaN(phoneCode)){
      return;
    }
    
    const parsedPhoneCode = Number.parseInt(phoneCode)
    const wilaya = data.find(w => w.phoneCodes.includes(parsedPhoneCode));
```

Reply:
[FIXED] no problem, I applied the change